### PR TITLE
fix: formatting and make seperate roles per lambda

### PR DESCRIPTION
**Pull request by rsmayda**
Description of changes:
- cosmetic: change from using `Ref:{}` to using `!Ref`
- cosmetic: Update MD files to use syntax highlighting
- fix: Update Install wording
- fix: Removed iamRoleStatements in favor or more specific FhirServerLambdaRole
- fix: Changed ES permissions from `*` to `ESHttp*`
- fix: Changed S3 permissions from `*` to `s3:*Object`; `s3:ListBucket`; `s3:DeleteObjectVersion`

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: serverless.yaml [hunk 1]
```diff
@@ -270,6 +195,7 @@ resources:
         DeletionPolicy: Retain
         UpdateReplacePolicy: Retain
         Properties:
+          TableName: '${self:custom.resourceTableName}'
```
Comment (carvantes) [inline serverless.yaml:198]: Changing the TableName requires replacement of the ddb table. Is that what we want?

** I'm sure it requires replacement when updating the name. I'm not 100% sure about what happens when changing from some table name to not specifying any name, but I'd expect it to be replaced.

Comment (rsmayda) [reply serverless.yaml:198]: Good call out I am actually just moving it from below in the properties; no functional change here. Old line was 284

File: serverless.yaml [hunk 2]
```diff
@@ -294,10 +219,11 @@ resources:
                 Value: 'daily'
               - Key: 'service'
                 Value: 'fhir'
-            - Ref: AWS::NoValue
+            - !Ref AWS::NoValue
       ExportRequestDynamoDBTable:
         Type: AWS::DynamoDB::Table
         Properties:
+          TableName: '${self:custom.exportRequestTableName}'
```
Comment (carvantes) [inline serverless.yaml:226]: Same thing here about renaming ddb tables

**Review by carvantes — DISMISSED** [review]

File: serverless.yaml (hunk 1 above)
Comment (carvantes) [reply serverless.yaml:198]: Right. My bad

File: INSTALL.md [hunk 3]
```diff
@@ -45,7 +46,7 @@ You can also use their abbreviations:
 sudo ./scripts/install.sh -r <REGION> -s <STAGE>
 ```
 
-#### Troubleshooting
+#### Linux or OSX Troubleshooting
```
Comment (nguyen102) [inline INSTALL.md]: Nitpick: I think OSX is just called macOS now.
https://en.wikipedia.org/wiki/OSX_(disambiguation)
`OS X was until September 2016 the name of an operating system family for Apple computers, thereafter known as macOS. `

**Review by nguyen102 — DISMISSED** [review]
Overall looks good. One small nitpick.

File: INSTALL.md (hunk 3 above)
Comment (rsmayda) [reply INSTALL.md]: a little TIL!

**Review by nguyen102 — DISMISSED** [review]

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**